### PR TITLE
Improve messages in AlreadyPaidPage

### PR DESCRIPTION
**Pull request by ahie**

### CODE REVIEW

File: src/components/pages/paymentPage/paymentError/AlreadyPaidPage.tsx [hunk 1]
```diff
@@ -2,13 +2,28 @@ import React from 'react';
...
+  }
+
   return (
     <NoticeTemplate
       id={'vene-already-paid-page'}
       titleText={t('page.payment_error.already_paid.title')}
-      message={<p>{t('page.payment_error.already_paid.message')}</p>}
+      message={<p>{t('page.payment_error.already_paid.paid_and_signed_message')}</p>}
     />
   );
```
Comment (TatuArvela) [inline src/components/pages/paymentPage/paymentError/AlreadyPaidPage.tsx:28]: To make this a bit shorter, I would do it like

```
  const message = isAdditionalProduct ? t('page.payment_error.already_paid.paid_message') : t('page.payment_error.already_paid.paid_and_signed_message');

  return (
    <NoticeTemplate
      id={'vene-already-paid-page'}
      titleText={t('page.payment_error.already_paid.title')}
      message={<p>{message}</p>}
    />
```

**Review by TatuArvela — APPROVED** [review]
LGTM! 👍 One suggestion added

File: src/components/pages/paymentPage/paymentError/AlreadyPaidPage.tsx [hunk 2]
```diff
@@ -2,13 +2,28 @@ import React from 'react';
...
+  if (isAdditionalProduct) {
+    return (
+      <NoticeTemplate
+        id={'vene-already-paid-page'}
+        titleText={t('page.payment_error.already_paid.title')}
+        message={<p>{t('page.payment_error.already_paid.paid_message')}</p>}
+      />
+    );
+  }
+
```
Comment (Nizar-Rahme) [inline src/components/pages/paymentPage/paymentError/AlreadyPaidPage.tsx:21]:
```suggestion
const messageKey = isAdditionalProduct
    ? 'page.payment_error.already_paid.paid_message'
    : 'page.payment_error.already_paid.paid_and_signed_message'
```

**Review by Nizar-Rahme — CHANGES_REQUESTED** [review]
One cosmetic comment

File: src/components/pages/paymentPage/paymentError/AlreadyPaidPage.tsx (hunk 1 above)
Comment (ahie) [reply src/components/pages/paymentPage/paymentError/AlreadyPaidPage.tsx]: 👍 Done

File: src/components/pages/paymentPage/paymentError/AlreadyPaidPage.tsx (hunk 2 above)
Comment (ahie) [reply src/components/pages/paymentPage/paymentError/AlreadyPaidPage.tsx]: Similar to @TatuArvela's comment :). Done.

File: src/components/pages/paymentPage/paymentError/AlreadyPaidPage.tsx (hunk 1 above)
Comment (Nizar-Rahme) [reply src/components/pages/paymentPage/paymentError/AlreadyPaidPage.tsx]: Oh, I had the same comment :D

**Review by Nizar-Rahme — APPROVED** [review]

**Pull request merged**